Description: This is a patch to install in /usr/bin instead of /local
Author: Mathieu Malaterre <malat@debian.org>
Forwarded: not-needed
Last-Update: 2012-03-05

--- a/config/Project.tmpl
+++ b/config/Project.tmpl
@@ -18,16 +18,16 @@ XCOMM ----------------------------------
    PRIVCONFIGDIR = PrivConfigDir
 
 #ifndef InstallBinDir
-#define InstallBinDir /usr/local/bin
+#define InstallBinDir /usr/bin
 #endif
 #ifndef InstallIncDir
-#define InstallIncDir /usr/local/include
+#define InstallIncDir /usr/include
 #endif
 #ifndef InstallLibDir
-#define InstallLibDir /usr/local/lib
+#define InstallLibDir /usr/lib
 #endif
 #ifndef InstallManDir
-#define InstallManDir /usr/local/man
+#define InstallManDir /usr/share/man
 #endif
 
           PATHSEP = /			/* for building filenames */
